tests/utils.go

```diff
@@ -106,6 +106,8 @@ const (
 	StartingVMInstance           = "Starting a VirtualMachineInstance"
 	WaitingVMInstanceStart       = "Waiting until the VirtualMachineInstance will start"
 	CouldNotFindComputeContainer = "could not find compute container for pod"
+	DeletingDataVolume           = "Deleting the DataVolume"
+	VerifyingGC                  = "Verifying DataVolume garbage collection"
```

Comment:
this 2 doesnt seem to belong here, should be in the same file as the helper functions they are used in